core/math/convex_hull.cpp

```diff
@@ -2252,19 +2253,62 @@ Error ConvexHullComputer::convex_hull(const Vector<Vector3> &p_points, Geometry3
 
 	r_mesh.vertices = ch.vertices;
 
+	// Tag which face each edge belongs to
+	LocalVector<int32_t> edge_faces;
+	edge_faces.resize(ch.edges.size());
+
+	for (uint32_t i = 0; i < ch.edges.size(); i++) {
+		edge_faces[i] = -1;
+	}
```

Comment:
I was going to say we had a `fill()` method for this, but I see it was added to 3.x in #60426 but not yet added to 4.x. Might be worth adding this at the same time as it is fairly common? :thinking:

Reply:
#60426 is a backport of a `master` PR so it should be in `master` too?

Reply:
> #60426 is a backport of a `master` PR so it should be in `master` too?

It wasn't included in the original as far as I can see. I don't think it is present in `LocalVector` in 4.x?